Examples/Tests/reduced_diagnostics_python/PICMI_inputs_reduced.py

```diff
@@ -0,0 +1,163 @@
+                                warpx_y_probe = 0.5,
+                                warpx_z_probe = 0.5,
+                                warpx_target_normal_x = 0,
+                                warpx_target_normal_y = 0,
+                                warpx_target_normal_z = 1,
+                                warpx_target_up_x = 0,
+                                warpx_target_up_y = 1,
+                                warpx_target_up_z = 0,
+                                warpx_detector_radius = 0.25,
+                                warpx_resolution = 10)
+FR_Max = picmi.ReducedDiagnostic(name='FR_Max',
+                                type='FieldReduction',
+                                intervals=200,
+                                warpx_reduction_type='Maximum',
+                                warpx_reduced_function='sqrt(Bx**2 + By**2 + Bz**2)'
+                                )
+FR_Min = picmi.ReducedDiagnostic(name='FR_Min',
+                                type='FieldReduction',
+                                intervals=200,
+                                warpx_reduction_type='Minimum',
+                                warpx_reduced_function='x*Ey*Bz'
+                                )
+FR_Integral = picmi.ReducedDiagnostic(name='FR_Integral',
+                                type='FieldReduction',
+                                intervals=200,
+                                warpx_reduction_type='Integral',
+                                warpx_reduced_function='"if(y > 0 and z < 0, '
+                                    + '0.5*((Ex**2 + Ey**2 + Ez**2)*epsilon0+(Bx**2 + By**2 + Bz**2)/mu0), 0)"'
+                                )
+
+reduced_diags = [EP, NP, EF, PP, PF, MF, MR, FP, FP_integrate, FP_line, FP_plane, FR_Max,FR_Min,FR_Integral]
+
+smoother = picmi.BinomialSmoother()
+# Electromagnetic solver
+solver = picmi.ElectromagneticSolver(
+    grid=grid,
+    method='Yee',
+    cfl=0.9999,
+    source_smoother=smoother
+    )
+
+# Diagnostics
+diag_field_list = ['B', 'E', 'J', 'rho', 'rho_electrons', 'rho_protons']
+field_diag = picmi.FieldDiagnostic(
+    name='diag1',
+    grid=grid,
+    period=200,
+    data_list=diag_field_list,
+    write_dir='diags')
+
+# Set up simulation
+sim = picmi.Simulation(
+    solver=solver,
+    max_steps=max_steps,
+    # max_time=max_time,
+    verbose=2,
+    particle_shape='linear',
+    warpx_charge_deposition_algo='standard',
+    warpx_current_deposition_algo='esirkepov',
+    warpx_field_gathering_algo='energy-conserving',
+    warpx_particle_pusher_algo='vay')
+
+for species in species_list:
+    sim.add_species(
+        species,
+        layout=picmi.GriddedLayout(grid=grid, n_macroparticle_per_cell=[1,1,1]))
+
+# Add diagnostics
+sim.add_diagnostic(field_diag)
+for rd in reduced_diags:
+    sim.add_reduced_diagnostic(rd)
+
+sim.initialize_inputs()
+sim.initialize_warpx()
+
+sim.write_input_file('inputs_PICMI_reduced')
```

Comment:
This is also not needed since nothing will be done with the `inputs_PICMI_reduced` file.